hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -369,6 +369,9 @@ private boolean onDeltaSyncShutdown(boolean error) {
     @Parameter(names = {"--help", "-h"}, help = true)
     public Boolean help = false;
 
+    @Parameter(names = {"--retry-last-pending-clustering", "-rc"}, description = "Retry last pending clustering plan before writing to sink.")
```

Comment:
Can we fix description that this applies only for inline clustering.

Reply:
may be, we can even name the variable coveying the info
--retry-last-pending-inline-clustering